### DISCUSSION TITLE
Face paints in makeup section are discolored.

### DISCUSSION THREAD

**Discussion by OrbitalVSAT** [MAFINS/MenyooSP#572] [Ideas]
Good day,

Before I updated Menyoo to its latest version, the face paints were fine. There was no discoloration until I updated to the latest version 1.8.1 the face paints looked blacked out. Additionally, when I make a new MP character the face paints seem to work again until I save them within the outfit files and it gets discolored again.

**Comment by GennosukeDono** [comment]
> Good day,
> 
> Before I updated Menyoo to its latest version, the face paints were fine. There was no discoloration until I updated to the latest version 1.8.1 the face paints looked blacked out. Additionally, when I make a new MP character the face paints seem to work again until I save them within the outfit files and it gets discolored again.

You're not getting discolored makeup, you're seeing them with a out of place back tint over the makeup layer.

Version 1.7.6 introduced the "feature" of adding a color select option for the makeups (which actually ruins most of them, vanilla or modded, since they are not designed to work that way), only the 0 value gets treated as 1 (black) everytime you load an outfit, thus making makeu useless.

You can "fix" the issue by rolling back to version 1.7.5, but what we need is an option to disable that "great idea".
